File: MPI/mpicheck/tests/functions/1.0/MPI_Graph_neighbors_count.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    int *var_2;
    int ret;
    /* calls */
    ret = MPI_Graph_neighbors_count(var_0, var_1, var_2);
    ret = PMPI_Graph_neighbors_count(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/1.0/MPI_Graph_neighbors_count.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_graph_neighbors_count(var_0, var_1, var_2, var_3)
       call pmpi_graph_neighbors_count(var_0, var_1, var_2, var_3)
        end program main
    